src/argosim/plot_utils.py

```diff
@@ -291,3 +292,76 @@ def plot_uv_hist(baselines, bins=20, output_folder=None):
         plt.show()
 
     return baseline_hist
+
+
+def plot_sll_visualization(beam_masked, sll_db, zoom):
```

Comment:
This function is not necessary, we can plot the masked beam with the `plot_sky` function.